packages/mds-policy-service/tests/validators.spec.ts

```diff
@@ -436,4 +437,145 @@ describe('Tests Policy Intents', () => {
       expect(validatePermittedParkingPolicyIntent(validIntent)).not.toBeFalsy()
     })
   })
+
+  describe('Test ProviderRebalancingZone intent', () => {
+    it('passes a valid intent', () => {
+      const validIntent = {
+        intent_type: 'provider_rebalancing_zone',
+        policy_fields: DEFAULT_POLICY_USER_FIELDS,
+        rule_fields: [
+          { ...DEFAULT_RULE_FIELDS, maximum: 100, states: {} },
+          { ...DEFAULT_RULE_FIELDS, maximum: 50, states: {} },
+          {
+            ...DEFAULT_RULE_FIELDS,
+            geographies: [uuid(), uuid()],
+            maximum: 0,
+            states: {
+              available: ['provider_drop_off'],
+              elsewhere: ['provider_drop_off'],
+              non_operational: ['provider_drop_off'],
+              on_trip: ['provider_drop_off'],
+              removed: ['provider_drop_off'],
+              reserved: ['provider_drop_off'],
+              unknown: ['provider_drop_off']
+            }
+          }
+        ]
+      }
+
+      expect(validateProviderRebalancingZonePolicyIntent(validIntent)).not.toBeFalsy()
+    })
+
+    it('throws if the states for the last rule are wrong', () => {
```

Comment:
I think that this test description is wrong - the states are correct here, but the maximum is not (should equal 0).